app.yaml

```diff
@@ -12,6 +16,10 @@ handlers:
 - url: /manual
   static_dir: src/manual
 
+- url: /images/(.*\.(bmp|gif|ico|jpeg|jpg|png))
```

Comment:
why use regex and not just upload the static files as is done already for manual ?